syntax = "proto3";

message EaveUserAction {
  message Action {
    string platform = 1;
    string name = 2;
    string description = 3;
    string opaque_params = 4;
    string eave_user_id = 5;
    int64 user_ts = 6;
  }

  Action action = 1;
  string message_source = 2;
}